demo/src/demoApp.js

```diff
@@ -1,38 +1,8 @@
 import {Navigation} from 'react-native-navigation';
-import {Typography, Constants, Colors, Assets, Button, Badge, ThemeManager, Text} from 'react-native-ui-lib'; //eslint-disable-line
 import * as Animatable from 'react-native-animatable';
-import './screens';
+import {Constants, Assets, Colors, Typography} from 'react-native-ui-lib'; //eslint-disable-line
+import {registerScreens} from './screens';
 
-// Typography.loadTypographies({
```

Comment:
keep this in comment, it's helpful to show people who clone this repo how to load colors, typos and how to use the theme manager